mindjunkies/lecture/tests/test_views.py

```
@@ -1,9 +1,29 @@
 import pytest
 from django.urls import reverse
 from model_bakery import baker
+from django.utils import timezone
+from datetime import timedelta
+from django.core.exceptions import ValidationError
```

Ruff (0.8.2)

`django.core.exceptions.ValidationError` imported but unused

Remove unused import: `django.core.exceptions.ValidationError`

(F401)